searchcore/src/tests/proton/attribute/attributeflush_test.cpp

```diff
@@ -161,11 +161,11 @@ UpdaterTask::run()
             }
             ia.update(j, i);
         }
-        ia.commit(i-1, i); // save i as last sync token
+        ia.commit(CommitParam(i-1, i)); // save i as last sync token
```

Comment:
i will be interpreted as bool here. Consider adding a third argument `false` to ensure i is considered to be a serial number.

Reply:
Ouch, then we have some very bad signatures here. Must fix.

Reply:
Fixed, PTAL